custom_components/healthchecksio/binary_sensor.py

```diff
@@ -80,6 +80,6 @@ def is_on(self):
         return self._status
 
     @property
-    def device_state_attributes(self):
+    def extra_state_attributes(self):
```

Comment:
From which Home Assistant version are `extra_state_attributes` valid?

Reply:
From 2021.12. I'm full of warnings from 10+ custom integrations

Reply:
Then this can not be merged.
No one can use it.

Reply:
You mean until next Core is released?

Reply:
At least, but probably closer to 2022.4

Reply:
It was mostly in preparation ahead of the release. But if left till 2022.4 you'd be leaving logging warnings for months.

Reply:
True, but that's better than a non working integration for those who don't/can't upgrade home assistant

Reply:
Although I'm now seeing this [#47304](https://github.com/home-assistant/core/pull/47304 ) so perhaps I was wrong about which version onwards this is valid. Seems like this has been valid since 2021.4? 😮

Reply:
That sounds way better :+1: